### PR TITLE
Implemented AnswerForm

### PR DESCRIPTION
**Pull request by davidtso1219**

### CODE REVIEW

File: src/AnswerForm.test.ts [hunk 1]
```diff
@@ -13,6 +11,10 @@ describe.skip('AnswerForm', () => {
 
   beforeEach(() => {
     answerForm = document.createElement('answer-form');
+    document.body.append(answerForm);
+
+    expect(answerForm).toBeTruthy();
+    if (answerForm == null) return; // TODO: research this
```
Comment (JohnTso) [inline src/AnswerForm.test.ts:17]: Can the const test variable be added inside the BeforeEach function?

File: src/AnswerForm.test.ts [hunk 2]
```diff
@@ -22,7 +24,9 @@ describe.skip('AnswerForm', () => {
     if (_button != null) button = _button;
     expect(button).toBeTruthy();
 
-    document.body.append(answerForm);
+    const _input = answerForm.querySelector('input');
+    if (_input != null) input = _input;
+    expect(input).toBeTruthy();
   });
 
   afterEach(() => {
```
Comment (JohnTso) [inline src/AnswerForm.test.ts:32]: Also taking the remove test here would be better maybe?

File: src/AnswerForm.ts [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+  static HTML = `
+  <form>
+    <fieldset>
+      <legend>Put Your Answer Below ⬇️</legend>
+      <input
+        type="text"
+        id="user-answer"
+        name="user-answer"
+        value=""
+        placeholder="your answer 📝"
```
Comment (JohnTso) [inline src/AnswerForm.ts:12]: Add a required attribute here

File: src/AnswerForm.test.ts [hunk 4]
```diff
@@ -1,9 +1,7 @@
-import type AnswerForm from './AnswerForm';
+import AnswerForm from './AnswerForm';
 
-import './AnswerForm';
-
-describe.skip('AnswerForm', () => {
```
Comment (JohnTso) [inline src/AnswerForm.test.ts:5]: Add another test for empty user answer

File: src/AnswerForm.test.ts [hunk 5]
```diff
@@ -33,50 +37,107 @@ describe.skip('AnswerForm', () => {
...
+  it('should remove event listeners correctly', () => {
+    const test = document.createElement('answer-form');
+    document.body.appendChild(test);
+
+    test.handleSubmit = jest.fn();
+    test.addEventListeners();
+    test.submit();
+
+    expect(test.handleSubmit).toHaveBeenCalled();
+
```
Comment (JohnTso) [inline src/AnswerForm.test.ts:74]: should add remove event listener here

**Review by JohnTso — APPROVED** [review]
LGTM!
